src/citrine/resources/predictor.py

```diff
@@ -596,3 +610,18 @@ def retrain_stale(self, uid: Union[UUID, str], *, version: Union[int, str]) -> G
         result in an error.
         """
         return self._versions_collection.retrain_stale(uid, version=version)
+
+    def rename(self,
+               uid: Union[UUID, str],
+               *,
+               version: Union[int, str],
+               name: str,
+               description: str) -> GraphPredictor:
```

Comment:
This caused me to realize the `/rename` endpoint requires both fields, which seems clunky and unlikely to be what users want. So I wrote up a ticket to make them optional: https://citrine.atlassian.net/browse/PLA-12705

Reply:
I agree